keras_contrib/applications/densenet.py

```diff
@@ -341,10 +351,14 @@ def __transition_block(ip, nb_filter, compression=1.0, dropout_rate=None, weight
                            beta_regularizer=l2(weight_decay))(ip)
     x = Activation('relu')(x)
     x = Convolution2D(int(nb_filter * compression), 1, 1, init="he_uniform", border_mode="same", bias=False,
-                      W_regularizer=l2(weight_decay))(x)
+                      W_regularizer=l2(weight_decay), dilation_rate=dilation_rate)(x)
     if dropout_rate:
         x = Dropout(dropout_rate)(x)
-    x = AveragePooling2D((2, 2), strides=(2, 2))(x)
+
+    if pooling == "avg":
+        x = AveragePooling2D((2, 2), strides=(2, 2))(x)
+    elif pooling == "max":
+        x = MaxPooling2D((2, 2), strides=(2, 2))(x)
 
```

Comment:
Handle pooling=None case by removing elif and instead use else: (default to average pooling by using if pooling == 'max')